src/components/BackOffice/CloseWorkOrders/index.js

```diff
@@ -88,7 +88,24 @@ const CloseWorkOrders = () => {
     return newErrors
   }
 
-  const handleSubmit = (event) => {
+  const renderConfirmationModal = () => {
+    if (showDialog) {
+      return (
+        <ConfirmationModal
+          title={'Permanently close Work Orders?'}
+          showDialog
+          setShowDialog={setShowDialog}
+          modalText={`The status of the selected Work Orders will change to "${
```

Comment:
Work Orders is capitalised throughout - is there a reason for this? It's not a proper noun.

Reply:
I agree it shouldn't be, but I've capitalised it to keep it consistent with the rest of the solution. Happy to change it everywhere to "work orders"

Reply:
yeah change it to no caps

Reply:
Will do

Reply:
Done